### PR TITLE
More explosion sizes for single tanks (NOT GREATER)

### PR DESCRIPTION
**Pull request by MacBlaze1**
Added more explosion sizes for single tanks, making them more useful without increasing explosion size(not tested)

## About The Pull Request

Changed strength threshold for the maximum explosion for single tanks from 1 to 2. 
Added less powerful explosion for strength 1 and above.
The new explosion is about a 1/2/3 or 1/2/4
## Why It's Good For The Game

Makes single tanks more intricate and customizable without power creeping by increasing explosion size. 

## Changelog
:cl:
add: New explosion for the strength of 1 and above, weaker than the maximum explosion right now.
tweak: Made strength 2 the threshold for the maximum explosion for single tanks.

/:cl:

Edit: Put the bomb change in the wrong temperature threshold and strength divisor, oops
Edit: Gave more explaination
Edit: Fixed actual size, wasn't consistent with lower or higher explosions

### CODE REVIEW

**Comment by nemvar** [comment]
wouldn't it be neat if some actual math was behind this instead of magic numbers... what am I talking... well guess this is inoffensive enough.

**Comment by Whoneedspacee** [comment]
> wouldn't it be neat if some actual math was behind this instead of magic numbers... what am I talking... well guess this is inoffensive enough.

just do normal tanks then lol

File: code/modules/assembly/bomb.dm [hunk 1]
```diff
@@ -166,7 +168,7 @@
 	else if(bomb_mixture.temperature > (T0C + 250))
 		strength = (fuel_moles/20)
 
-		if(strength >=1)
+		if (strength >=1)
```
Comment (Tlaltecuhtli) [inline code/modules/assembly/bomb.dm]: why

Comment (MacBlaze1) [reply code/modules/assembly/bomb.dm]: Oh jesus christ, thats a random bad that I did.

**Comment by actioninja** [comment]
I don't like the existence of singlecaps but if anything this just means less people will be making the max power of them.

**Comment by Whoneedspacee** [comment]
its not singlecaps though

**Pull request merged**